### PR TITLE
Updated for for arbitrary command line arg ordering

### PR DESCRIPTION
**Pull request by davidbrownellWork**

### CODE REVIEW

**Review by peterjc123 — APPROVED** [review]
LGTM

**Review by orionr — CHANGES_REQUESTED** [review]
Nicely done! One code change needed since it changed underneath you. Can you rebase and fix? Thanks.

File: tools/build_pytorch_libs.bat [hunk 1]
```diff
@@ -14,54 +16,70 @@ set LDFLAGS=/LIBPATH:%INSTALL_DIR%/lib
...
+
+: Variable defaults
+set /a USE_CUDA=0
+set /a USE_ROCM=0
+set /a USE_NNPACK=0
+set /a USE_QNNPACK=0
+set /a USE_GLOO_IBVERBS=0
+
+set /a NO_NNPACK=1
+set /a NO_MKLDNN=1
```
[the remark this answers is not in the document]
Comment (davidbrownellMS) [reply tools/build_pytorch_libs.bat]: It doesn't look like NO_MKLDNN is used in the code after this change. I've updated the docs in setup.py.

**Comment by davidbrownellMS** [comment]
I've rebased and added a change in setup.py to document USE_MKLDNN.

**Review by orionr — APPROVED** [review]

**Pull request closed without merging**